### PR TITLE
slash command ephemeral message fix

### PR DESCRIPTION
**Pull request by yofukashino**

### CODE REVIEW

File: src/renderer/apis/commands.ts [hunk 1]
```diff
@@ -26,10 +26,10 @@ interface CommandsAndSection {
   commands: Map<string, AnyRepluggedCommand>;
 }
 
-void waitForModule<typeof User>(filters.bySource(".isStaffPersonal=")).then((User) => {
+void waitForModule<typeof User>(filters.bySource('"isStaffPersonal",')).then((User) => {
   RepluggedUser = new User({
     avatar: "replugged",
-    id: "replugged",
+    id: "69",
```
Comment (lexisother) [inline src/renderer/apis/commands.ts:32]: This change is no longer required. I got it working just by replacing the source match.

File: src/renderer/coremods/commands/index.ts [hunk 2]
```diff
@@ -270,7 +270,8 @@ async function injectProfileFetch(): Promise<void> {
     fetchProfile: (id: string) => Promise<void>;
   }>("fetchProfile");
   injector.instead(mod, "fetchProfile", (args, res) => {
-    if (args[0] === "replugged") {
+    //69 is replugged user id
+    if (args[0] === "69") {
```
Comment (lexisother) [inline src/renderer/coremods/commands/index.ts:274]: No longer required.

**Review by lexisother — DISMISSED** [review]

File: src/renderer/apis/commands.ts (hunk 1 above)
Comment (YofukashiNo) [reply src/renderer/apis/commands.ts:32]: should i close the pr then?

Comment (lexisother) [reply src/renderer/apis/commands.ts:32]: No? The source match change is still required.

Comment (lexisother) [reply src/renderer/apis/commands.ts:32]: Yes, I also saw that, but you have to consider that was quite a while ago. I tested it *today* without replacing the `id` and it worked fine. However, I didn't test clicking the fake user. Is that where the issue lies?

Comment (YofukashiNo) [reply src/renderer/apis/commands.ts:32]: yes

**Review by lisekilis — CHANGES_REQUESTED** [review]
Change from `69` to `826980768571716968` aka REPLUGGED in ascii (with the separators removed)
... also fix prettier lol

**Review by colin273 — APPROVED** [review]

**Pull request merged**